django/utils/http.py

```diff
@@ -315,8 +319,9 @@ def _is_safe_url(url, host):
     # URL and might consider the URL as scheme relative.
     if unicodedata.category(url[0])[0] == 'C':
         return False
+    valid_schemes = ['https'] if secure else ['http', 'https']
```

Comment:
the only problem with check I guess that urls like `//testserver/` will be considered as secure which is not I guess